HW9.3/HW9.3.cpp

```diff
@@ -0,0 +1,32 @@
+﻿#include <cstring>
+#include <iostream>
+#include <cctype>
+
+
+bool isPalindrom(const char str[], int lineSize) 
+{
+    int length = strlen(str);
+    int i = 0, j = length-1 ;
+    while (i < j) 
+    {
+        if ((str[i]) != (str[j])) 
+        {
+            std::cout << "error";
+            return false;
+        }
+        i++;
+        j--;
+    }
+    std::cout << "success\n" << str;
+    return true;
+}
+
+int main() {
+    const int lineSize = 25;
+    char line[lineSize];
+    std::cin.getline(line, lineSize);
+    isPalindrom(line, lineSize);
```

Comment:
add if here if you want some message about palindrome:
if (isPalindrom(line, lineSize)) cout << "success"; else cout << "error";